packages/graphql/eslint.config.js

```diff
@@ -0,0 +1,11 @@
+// @ts-check
```

Comment:
we also have a standard eslint config at the top which should validate your code automatically

Reply:
When you say automatically, do you mean I need to run `npm run lint`?

Reply:
yeah sorry, I meant as part of our comon CI but yeah `pnpm lint` from the top of the repo.

Most of our packages also have this
```
    "lint": "eslint . --max-warnings=0",
    "lint:fix": "eslint . --fix"
```